extensions/amp-ad-exit/0.1/config.js

```diff
@@ -165,10 +167,10 @@ function assertTarget(name, target, config) {
  * @return {string} The vendor's iframe URL
  */
 export function assertVendor(vendor) {
-  user().assert(ANALYTICS_CONFIG &&
-      ANALYTICS_CONFIG[vendor] &&
-      ANALYTICS_CONFIG[vendor]['transport'] &&
-      ANALYTICS_CONFIG[vendor]['transport']['iframe'],
+  user().assert(ANALYTICS_IFRAME_TRANSPORT_CONFIG &&
+      ANALYTICS_IFRAME_TRANSPORT_CONFIG[vendor] &&
+      ANALYTICS_IFRAME_TRANSPORT_CONFIG[vendor]['transport'] &&
+      ANALYTICS_IFRAME_TRANSPORT_CONFIG[vendor]['transport']['iframe'],
   'Unknown vendor: ' + vendor);
```

Comment:
Can we make this error message a little more helpful?  `Unknown or invalid vendor ${vendor}, note that vendor must use transport: iframe`

Reply:
Done